src/curated-corpus/components/ExistingProspectCard/ExistingProspectCard.test.tsx

```diff
@@ -164,4 +165,39 @@ describe('The ExistingProspectCard component', () => {
 
     expect(scheduleButton).toBeInTheDocument();
   });
+
+  it('should render schedule history component', async () => {
+    // creating a prospect with an schedule history
+    const prospectWithScheduleHistory = {
+      ...prospect,
+      approvedCorpusItem: {
+        ...prospect.approvedCorpusItem!,
+        authors: [{ name: 'Marie Curie', sortOrder: 1 }],
+        scheduledSurfaceHistory: [
+          {
+            createdBy: 'ad|Mozilla-LDAP|aperson',
+            externalId: 'bogus-external-id',
+            scheduledDate: '2022-02-08',
+            scheduledSurfaceGuid: ScheduledSurfaces[0].guid,
+          },
+        ],
+      },
+    };
+
+    render(
+      <MemoryRouter>
+        <ExistingProspectCard
+          item={prospectWithScheduleHistory.approvedCorpusItem}
+          onSchedule={onSchedule}
+        />
+      </MemoryRouter>
+    );
+
+    // the button when clicked shows us the recent scheduled runs
+    const recentScheduledRunsButton = screen.getByRole('button', {
+      name: /view recent scheduled runs/i,
+    });
+
+    expect(recentScheduledRunsButton).toBeInTheDocument();
```

Comment:
For clarity, it would be good to add a comment here that clicking on the button and making sure we get all data displayed on the page is tested in unit tests for the `ScheduleHistory` component.

Reply:
Good call, added a top level comment in that test!